packages/contracts/src/diamond/libraries/LibCreditNFTManager.sol

```diff
@@ -153,10 +152,10 @@ library LibCreditNftManager {
         );
 
         // we burn user's dollars.
-        LibDollar.burn(msg.sender, amount);
+        IERC20Ubiquity(s.dollarTokenAddress).burnFrom(msg.sender, amount);
         // mint Credit
         UbiquityCreditToken creditToken = UbiquityCreditToken(
-            LibAppStorage.appStorage().creditTokenAddress
+            s.creditTokenAddress
```

Comment:
Not a fan of one letter identifiers. Why not just call it storage or store

Reply:
storage can't be used as a variable name as it is a keyword in solidity  
will rename to store then